---
description: "Saiba mais sobre: a ramificação de ' Finally ' não é válida"
title: Ramificação saindo de uma 'Finally' é inválida
ms.date: 07/20/2015
f1_keywords:
- bc30101
- vbc30101
helpviewer_keywords:
- BC30101
ms.assetid: 16a0dc29-3657-4373-b77f-38f3cb80e6c9
ms.openlocfilehash: 764ef32d60112a3c1c6365cf9e2773643a83ed41
ms.sourcegitcommit: 10e719780594efc781b15295e499c66f316068b8
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 02/14/2021
ms.locfileid: "100478549"
---
# <a name="branching-out-of-a-finally-is-not-valid"></a>Ramificação saindo de uma 'Finally' é inválida

Uma `GoTo` instrução dentro de um `Finally` bloco ramifica fora do bloco. Não é válido ramificar para dentro ou para fora de `Catch` um `Finally` bloco ou.  
  
 **ID do erro:** BC30101  
  
## <a name="to-correct-this-error"></a>Para corrigir este erro  
  
- Remova a `GoTo` instrução e considere implementar a lógica do programa com estruturas de controle de decisão ou loop.  
  
## <a name="see-also"></a>Consulte também

- [Instrução Try...Catch...Finally](../language-reference/statements/try-catch-finally-statement.md)
- [Instrução GoTo](../language-reference/statements/goto-statement.md)
- [Fluxo de Controle](../programming-guide/language-features/control-flow/index.md)
